blank-canvas-blocks/experimental-theme.json

```diff
@@ -3,6 +3,10 @@
 		{
 			"name": "header",
 			"area": "header"
+		},
+		{
+			"name": "footer",
+			"area": "footer"
```

Comment:
do we need one for the navigation or is that not a thing?

Reply:
TY.  I believe we do, yes.